Dockerfile

```diff
@@ -19,16 +24,10 @@ COPY .git .git
 
 # Build
 RUN git config --get remote.origin.url > /tmp/GIT_REPO && \
```

Comment:
Can we go with `ARG` strategy also for `GIT_REPO` and `BUILD_DATE`?

Reply:
Yes, I think I can do it! :smile:

Reply:
It would be great, thanks!